tests/test_invalid_certs.rs

```diff
@@ -0,0 +1,58 @@
+#[cfg(feature = "tls")]
```

Comment:
`tests` in `Cargo.toml` do not support `required-features`?

Reply:
Yes it does, for some reason I assumed it didn't.